### PR TITLE
Skeleton/hamburgermenu

### LINKED ISSUE #65: Implement hamburger view containing account, settings, and other buttons

### PR DESCRIPTION
**Pull request by danielpgp1012**
implement hamburger menu with home, account, settings, about and share tabs. 
Keep 2 bottom tabs: map and favor list view

Closes #65

### CODE REVIEW

File: app/src/main/java/ch/epfl/favo/MainActivity.java [hunk 1]
```diff
@@ -1,79 +1,209 @@
...
-                  }
-                case 2:
-                  {
-                    tab.setIcon(R.drawable.ic_face_24px);
-                    break;
-                  }
-              }
-            })
-        .attach();
+  @Override
```
[the remark this answers is not in the document]
Comment (danielpgp1012) [reply app/src/main/java/ch/epfl/favo/MainActivity.java]: Refactored successfully

**Comment by muneebazam** [comment]
I would recommend creating a folder for all the hamburgermenu fragments inside the view/ folder. It will help us keep things clean in the future as we add more files, but otherwise lgtm

**Comment by muneebazam** [comment]
you can do it as a part of your next PR though, the functionality looks good

File: app/src/androidTest/java/ch/epfl/favo/MainActivityTest.java [hunk 2]
```diff
@@ -12,46 +11,283 @@
...
+
+        getInstrumentation().waitForIdleSync();
+
+        //check that menu drawer is displayed
+        onView(withId(R.id.nav_view))
+                .check(matches(isDisplayed()));
+    }
+    @Test
+    public void testMenuDrawerCanBeLaunchedFromFavorsView(){
+        //Click on map tab
```
Comment (huajian1069) [inline app/src/androidTest/java/ch/epfl/favo/MainActivityTest.java]: click on favour tab

File: app/src/androidTest/java/ch/epfl/favo/MainActivityTest.java [hunk 3]
```diff
@@ -12,46 +11,283 @@
...
+    public void testSettingsTabIsLaunched(){
+
+        //Click on menu tab
+        onView(withId(R.id.hamburger_menu_button))
+                .check(matches(isDisplayed()))
+                .perform(click());
+
+        getInstrumentation().waitForIdleSync();
+
+        //Click on account icon
```
Comment (huajian1069) [inline app/src/androidTest/java/ch/epfl/favo/MainActivityTest.java]: click on setting icon

File: app/src/androidTest/java/ch/epfl/favo/MainActivityTest.java [hunk 4]
```diff
@@ -12,46 +11,283 @@
...
+    public void testAboutTabIsLaunched(){
+
+        //Click on menu tab
+        onView(withId(R.id.hamburger_menu_button))
+                .check(matches(isDisplayed()))
+                .perform(click());
+
+        getInstrumentation().waitForIdleSync();
+
+        //Click on account icon
```
Comment (huajian1069) [inline app/src/androidTest/java/ch/epfl/favo/MainActivityTest.java]: click on about icon

File: app/src/androidTest/java/ch/epfl/favo/MainActivityTest.java [hunk 5]
```diff
@@ -12,46 +11,283 @@
...
+    public void testShareTabIsLaunched(){
+
+        //Click on menu tab
+        onView(withId(R.id.hamburger_menu_button))
+                .check(matches(isDisplayed()))
+                .perform(click());
+
+        getInstrumentation().waitForIdleSync();
+
+        //Click on account icon
```
Comment (huajian1069) [inline app/src/androidTest/java/ch/epfl/favo/MainActivityTest.java]: click on share icon

**Pull request merged**